react-with-json-logic/.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
should I configure [restyled](https://restyled.io/)? I am using it in another project and it makes PR easier to compare (its also compatible with prettier) :)

Reply:
Just saw your comment. I added two scripts: "npm run format" to style and "npm run lint" to check and I added it to the CI job. Of course, it does not add an alternative PR.

If you know this tool and it works well, I would say: go for it.